acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/UpcheckOpenApiValidationTest.java

```diff
@@ -12,15 +12,27 @@
  */
 package tech.pegasys.eth2signer.tests;
 
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.text.IsEqualIgnoringCase.equalToIgnoringCase;
+
 import tech.pegasys.eth2signer.dsl.signer.SignerConfigurationBuilder;
 
 import org.junit.jupiter.api.Test;
 
-public class UpcheckAcceptanceTest extends AcceptanceTestBase {
-
+public class UpcheckOpenApiValidationTest extends AcceptanceTestBase {
   @Test
-  public void upcheckOnCorrectPortRespondsWithOK() {
+  void testGetUpcheck() {
```

Comment:
I prefer the original test name "upcheckOnCorrectPortRespondsWithOK" as it is more descriptive